packages/core/src/browser/quick-input/quick-input-contribution.ts

```diff
@@ -0,0 +1,23 @@
+/********************************************************************************
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+export const QuickInputContribution = Symbol('QuickInputContribution');
+/**
+ * The quick input contribution should be implemented to register custom quick input handler.
+ */
+export interface QuickInputContribution {
```

Comment:
This should be called `QuickAccessContribution`

Reply:
You are right. Fixed